docs: remove ref to no longer existing `test.ts` (#51462)
angular/angular: aio/content/guide/testing-components-scenarios.md

#### Async test with `fakeAsync()`

To use `fakeAsync()` functionality, you must import `zone.js/testing` in your test setup file.

> If you created your project with the Angular CLI, `zone-testing` has already been added for you as part of Karma's [builder](https://angular.io/guide/glossary#builder) config.

Reviewer: ```suggestion
If you created your project with the Angular CLI, `zone-testing` has already been added for you.
```

Author: Thanks! Done in bb8a0d7490b215414cdecf68a750c67197f2b89d (force pushed changes)

> If you created your project with the Angular CLI, don't worry: `zone-testing` has already been added for you as part of Karma 's [builder](https://angular.io/guide/glossary#builder).

Reviewer 2: We should mention that the CLI adds it in the polyfills entry of the angular config. (And probably we should give this advice instead of importing it in the file, instead of what's suggested before)

Author: Yeah, actually if this test setup file is no longer a thing in new Angular projects maybe shouldn't be mentioned at all. 

However I feel that talking about builders and their polyfills can be a bit advanced for this kind of guide which I understand targets someone who just wants to get started testing components. 

Maybe it could be added as a note? (this would replace the first line too)

```suggestion
<div class="alert is-helpful">
**NOTE**: <br />

To use `fakeAsync()` functionality, you must import `zone.js/testing`. If you created your project with the Angular CLI, don't worry: `zone-testing` has already been added for you as part of Karma 's [builder](https://angular.io/guide/glossary#builder).
</div>
```

#### Async test with `waitForAsync()`

To use `waitForAsync()` functionality, you must import `zone.js/testing` in your test setup file.

> If you created your project with the Angular CLI, `zone-testing` has already been added for you as part of Karma's [builder](https://angular.io/guide/glossary#builder) config.

Reviewer: ```suggestion
If you created your project with the Angular CLI, `zone-testing` has already been added for you.
```

Author: Thanks! Done in bb8a0d7490b215414cdecf68a750c67197f2b89d (force pushed changes)

> If you created your project with the Angular CLI, don't worry: has `zone-testing` already been added for you as part of Karma 's [builder](https://angular.io/guide/glossary#builder).

Reviewer 2: The "don't worry" feels unnecessary. 
```suggestion
If you created your project with the Angular CLI `zone-testing` already been added for you as part of Karma 's [builder](https://angular.io/guide/glossary#builder) config.
```

Author: absolutely agree! just force-pushed changes

c222c0c3c4174f27ad12f8c36823214930f11982

thanks :)
